code/game/objects/items/rogueweapons/melee/blunt.dm

```diff
@@ -186,9 +186,8 @@
 	blade_class = BCLASS_PICK
 	attack_verb = list("picks", "impales")
 	hitsound = list('sound/combat/hits/blunt/metalblunt (1).ogg', 'sound/combat/hits/blunt/metalblunt (2).ogg', 'sound/combat/hits/blunt/metalblunt (3).ogg')
-	chargetime = 14
+	swingdelay = 14
```

Comment:
> change this to 10 like the smash intent, otherwise this is way too slow to use. The charge time was set to 14 specifically so that it was worse than smash intent with the counter play that could ready it up for an opportune strike (which would most likely get blocked anyway)

Yeah that's the idea, it's the same reason why the dagger pick intent had it. Because the benefit of completely going through any armor shouldn't come at the cost of simply attacking slower, but making it hard or situational to land.

Also, why would you like to change the swing delay to the same as smash intent's if you yourself tried to set it so it's specifically worse than the smash intent?

> Same for damage factor, set that back to 1.1 if this is going to be the only viable way of penetrating plate armor completely. You know this will just make warhammers the meta choice right?

I'd like to ask why you think it would make them a meta choice and also make them too slow to use at the same time, seems a bit contradicting doesn't it?
Oh and yeah, seems fair enough to raise it back to 1.1.

Reply:
>Because now there isn't a smash intent for it to be worse than, the issue with such a long wait time is that people can VERY easily dodge it, even with a delay of 10 you could dodge it easily. no reason to make it so appallingly bad

The smash intent it's still there, and it's still *pretty* good at damaging people.

>Not at all, because you've nerfed everything else with smash intent but seemed to leave this one method of penning armor alone, you can't make the decision and then call me contradictory for merely pointing it out.

Well let's point out that i didn't leave it alone, i quite litterally made it considerably hard to land as you've said.

What i meant is that it isn't going to be the only viable way of penetrating armor, like you said, it's considerably hard to land with the new delay **and** smash is still good at penetrating armor, and it still deals a very good amount of damage, and goes through most common armors, it just simply doesn't completely bypass plate armor.

What i mean is that the smash intent wasn't nerfed enough to now be completely useless, it's just that now it's slightly less damaging against people that are wearing plated armors or heavy armors specifically made to resist blunt damage.

But i think i can say that neither of us can accurately say what will become meta unless it's at least tested for a bit.

And i'd like to point out that i didn't leave it alone, i quite literally made it considerably hard to land, as you've said. 
That was the whole point of the pick intent. Hard to land, very rewarding when it does.